server/models/Review.js

```diff
@@ -1,23 +1,32 @@
 const mongoose = require("mongoose");
 
-const reviewSchema = new mongoose.Schema(
-  {
-    user: {
-      firstName: String,
-      lastName: String,
-      profileImg: String,
-    },
-    rating: {
-      type: Number,
-      min: 1,
-      max: 5,
-      required: true,
-    },
-    message: String,
+const { Schema } = mongoose;
+const { ObjectId } = Schema;
+
+const reviewSchema = new mongoose.Schema({
```

Comment:
Can we add a field for a written review and not require it?

Reply:
Sure, I do that